src/renderer/flow-typed/bluebird.js

```diff
@@ -0,0 +1,3 @@
+declare module 'bluebird' {
```

Comment:
There's got to be a better way to do this. Can these be added to the `.flowconfig`?

Reply:
Have you checked if flow-typed has any of the type definitions?

https://github.com/flowtype/flow-typed

`yarn run flow-typed install bluebird`